packit_service/worker/jobs.py

```diff
@@ -483,6 +483,18 @@ def get_jobs_matching_event(self) -> List[JobConfig]:
             ):
                 jobs_matching_trigger.append(job)
 
+        if not jobs_matching_trigger and isinstance(
+            self.event, PullRequestCommentPagureEvent
+        ):
+            if (
+                job.type == JobType.koji_build
```

Comment:
personal preference: It would be awesome if you named this longer condition so that it's clear to a person what it means without a few extra seconds of thinking (yes, I am a lazy person :D). What about reusing the comment you posted bellow "A koji_build can be retriggered by a Pagure comment in a PR"... so something like `can_koji_build_be_retriggered`?